test/functional/p2p_tx_download.py

```
@@ -103,62 +97,125 @@ def test_inv_block(self):
         #   peer, plus
         # * the first time it is re-requested from the outbound peer, plus
         # * 2 seconds to avoid races
-        timeout = 2 + (MAX_GETDATA_RANDOM_DELAY + INBOUND_PEER_TX_DELAY) + (
-            GETDATA_TX_INTERVAL + MAX_GETDATA_RANDOM_DELAY)
-        self.log.info("Tx should be received at node 1 after {} seconds".format(timeout))
-        self.sync_mempools(timeout=timeout)
+        assert self.nodes[1].getpeerinfo()[0]['inbound'] == False
```

Flake8 (7.3.0)

[error] comparison to False should be 'if cond is False:' or 'if not cond:'

(E712)